index.html

```diff
@@ -55,11 +55,35 @@
       <div class="list">
         <div class="todo">
           <div class="todo-header">
-            <span>import</span>
-            <span>messages</span>
-            <span>due-date</span>
-            <span>creation-date</span>
-            <span>priority</span>
+            <span></span>
+            <span
+              class="colHeader"
+              data-column="message"
+              data-state="0"
```

Comment:
Could you please explain me how data-state works?

Reply:
It is for indicating to us if the column is order by ascendent or descendent